client/my-sites/marketing/traffic/index.js

```diff
@@ -123,9 +123,6 @@ const getFormSettings = partialRight( pick, [
 	'hide_smile',
 	'count_roles',
 	'roles',
-	'enable_header_ad',
-	'wordads_ccpa_enabled',
-	'wordads_ccpa_privacy_policy_url',
```

Comment:
I cannot witness any change in `https://wordpress.com/marketing/traffic/`. Were these unused?

Reply:
Yup, there are no longer used as of https://github.com/Automattic/wp-calypso/pull/51689 (which is a PR that I investigated in order to see how the settings were previously managed on Atomic/Jetpack sites).